### ISSUE TITLE
Still in development

### ISSUE THREAD

**Issue by keithmancuso** [stomita/react-lightning#1]
we've been looking for a sustainable way to do react development in lightning and are big fans of jsforce so fingers crossed

**Comment by ruslan-kurchenko** [comment]
Guys, are you still working on this project?
I'm very excited to see the progress on React in Lightning.
How can I help you, any contribution required?

**Comment by stomita** [comment]
@ruslan-kurchenko Yes I'm working on, and still believing this approach is neat. I'm expecting more investigation to be added to make stable this lib, but I'm currently sparing my time to the other project and do not have so much time to devote right now. It also should be considered the release of the next Lightning platform update (meaning LWC).
